sources/ingest/fetch/location.py

```diff
@@ -67,6 +67,11 @@ class OntologyObject:
     id: str
     label: LanguageString
 
+@dataclass
+class Image:
+    image_url: str
```

Comment:
Other data classes seem to use camelCase instead of snake_case.

Reply:
Using plain url now. It is true that there's same unwanted camelCasing in place now. Clean up to be done in another PR.